packages/beacon-node/src/metrics/metrics.ts

```diff
@@ -1,19 +1,21 @@
-import {ChainForkConfig} from "@lodestar/config";
-import {BeaconStateAllForks, getCurrentSlot} from "@lodestar/state-transition";
-import {Logger} from "@lodestar/utils";
+import {BeaconStateTransitionMetrics} from "@lodestar/state-transition";
+import {getMetrics} from "@lodestar/state-transition/src/metrics.js";
```

Comment:
this should not require to access `src` path

```suggestion
import {getMetrics} from "@lodestar/state-transition";
```

something like this should ideally be caught by the linter as it would break the beacon-node package if used as standalone package (ie. not executed as part of monorepo)